datasets/submit.sh

```diff
@@ -4,10 +4,7 @@
 # ./submit.sh COMMAND...
```

Comment:
I know the instructions for the flag options are in the README, but I think it would also be helpful to include it here